src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/ValueBox.scala

```diff
@@ -0,0 +1,84 @@
+package org.broadinstitute.dsde.workbench.leonardo.util
+
+/**
+  * A simple container to hold a value and make it safer to access and update it
+  * Copied from https://github.com/broadinstitute/dig-loam-stream/blob/master/src/main/scala/loamstream/util/ValueBox.scala
```

Comment:
^^ is a project I worked on where `ValueBox` was useful so I wanted to spread the love ❤️

Reply:
Cool - maybe it could go in workbench-libs?

Reply:
I agree. Will try to factor it out in a separate PR.